pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: floater
-description: A new Flutter package project.
+description: Flutter framework with helpers that floats on top of flutter to make flutter development simple.
```

Comment:
Suggestion:
A framework that floats over flutter as a higher level construct to provide architecture and structure for complex flutter apps.